### PR TITLE
feat: [CDS-82272]: Add OnEnableFreezeWindow type to freeze events

### PR DESCRIPTION
**Pull request by pankajsingh-harness**
#### Screenshots

**Before:**
Screenshot 2023-11-20 at 3 36 30 PM

**After:** 
Screenshot 2023-11-20 at 3 36 16 PM

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

### CODE REVIEW

**Comment by pankajsingh-harness** [comment]
trigger messagecheck

**Comment by pankajsingh-harness** [comment]
retrigger eslint

**Review by Sagar-Rajput7 — APPROVED** [review]

File: src/modules/72-freeze-windows/components/FreezeNotifications/FreezeEvents.tsx [hunk 1]
```diff
@@ -82,12 +97,15 @@ export const FreezeEvents = ({ nextStep, prevStepData }: StepProps<FreezeNotific
                   </Text>
                 )}
                 {eventItems.map(event => {
+                  const { label, value } = event
+                  const eventValue = value as string
```
Comment (samarth-sharma31) [inline src/modules/72-freeze-windows/components/FreezeNotifications/FreezeEvents.tsx:101]: can be simplified to ->
const {label, value: eventValue} = event

**Review by samarth-sharma31 — APPROVED** [review]

File: src/modules/72-freeze-windows/components/FreezeNotifications/FreezeEvents.tsx (hunk 1 above)
Comment (pankajsingh-harness) [reply src/modules/72-freeze-windows/components/FreezeNotifications/FreezeEvents.tsx:101]: @samarth-sharma31 it this case we can not typecase the value.

**Pull request merged**